src/js/overlay.js

```diff
@@ -74,7 +74,13 @@ const focusTrap = function(event) {
 	const tabKeyCode = 9;
 	const overlayFocusableElements = [].slice.call(
 		this.wrapper.querySelectorAll('button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])')
-	).filter(element => isVisible(element) && !element.disabled);
+	).filter(element => {
+		const elementVisible = isVisible(element);
+		const elementLabelVisible = element.labels && [].slice.call(element.labels).some(l => isVisible(l));
```

Comment:
We should comment this as it is something that we may not remember when coming back to this function in the future
```suggestion
		// Inputs for radio and checkboxes are visually hidden, 
		// so check the label visibility of inputs too when determining
		// whether to trap focus.
		const elementLabelVisible = element.labels && [].slice.call(element.labels).some(l => isVisible(l));
```

Reply:
Better. Thanks :)